### PR TITLE
Enable organize-imports for mtags3

### PR DESCRIPTION
**Pull request by dos65**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -171,6 +171,7 @@ Global / onLoad ~= { old =>
   old
 }
 Global / cancelable := true
+Global / excludeLintKeys += scalafixConfig
```
Comment (tgodzik) [inline build.sbt:174]: What does this do?

**Review by tgodzik — APPROVED** [review]
Och cool, I didn't think about it.

File: build.sbt (hunk 1 above)
Comment (dos65) [reply build.sbt:174]: sbt wrongly warns that `scalafixConfig` setting isn't used

Comment (bjaglin) [reply build.sbt:174]: I just opened https://github.com/scalacenter/sbt-scalafix/pull/233 to avoid that

**Pull request merged**